src/services/session-manager.ts

```diff
@@ -71,17 +71,17 @@ export class OnlineSessionManager implements SessionManager {
 
   private logOut() {
     this.redirect();
-    this.updateSession({ status: 'non-authenticated' });
+    // TODO: request Doppler to close the session
+    // TODO: Support two way of logout, redirecting or not
+    // this.updateSession({ status: 'non-authenticated' });
   }
 
   // TODO: move into a dependency
   private redirect() {
     const currentUrlEncoded = encodeURI(window.location.href);
     // TODO: only use redirect on login, not in logout
     const loginUrl = `${process.env.REACT_APP_API_URL}/SignIn/index?redirect=${currentUrlEncoded}`;
-    window.setTimeout(() => {
-      // Redirecting in a timeout in order to allow React to update UI
```

Comment:
Why removing this timeout now? seems to be neccesary, did you add another thing to fix react ui update?

Reply:
Well, I thought the same, but @fcoronelmakingsense was right.

If we redirect directly to Doppler when the session status is _not-autenticated_, we do not need to deal with that state. Is simpler. 

Of course, we will need to add a login in our side, so, it will be necessary to deal with _not-autenticated_, but that is another topic.

Reply:
I just cleaned a little more and deleted deprecated comments.